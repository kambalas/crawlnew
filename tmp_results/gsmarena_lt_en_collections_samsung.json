[
  {
    "name": "Samsung Galaxy Fold6 512GB",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold6 256GB",
    "price": "1288.99",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold6 256GB (Pažeista pakuotė)",
    "price": "1270.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold5 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 5G 512GB",
    "price": "1098.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 5G 512GB (Pažeista pakuotė)",
    "price": "1080.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 5G 512GB",
    "price": "995.98",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold5 256GB (Pažeista pakuotė)",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 5G 512GB (Pažeista pakuotė)",
    "price": "975.98",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 5G 256GB",
    "price": "970.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 5G 256GB",
    "price": "888.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy Flip6 512GB",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 5G 256GB (Damaged packaging)",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold4 5G 256GB",
    "price": "869.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 5G 256GB",
    "price": "813.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 5G 512GB",
    "price": "800.79",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 5G 256GB (Ekspozicinė prekė)",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S23 Ultra 5G 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy Flip6 256GB",
    "price": "781.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold4 256GB (Damaged Packaging)",
    "price": "759.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 5G 512GB (Used item)",
    "price": "749.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S23 Plus 5G 256GB (Pažeista pakuotė)",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold5 256GB (Ekspozicinė prekė)",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 5G 256GB",
    "price": "692.00",
    "error": false
  }
]