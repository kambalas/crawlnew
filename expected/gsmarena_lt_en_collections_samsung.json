[
  {
    "name": "Samsung Galaxy Fold6 512GB",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 5G 512GB Titanium Jet Black",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 5G 1TB",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 5G 1TB",
    "price": "1362",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold6 256GB",
    "price": "1288.99",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold6 256GB (Pažeista pakuotė)",
    "price": "1270",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 5G 512GB",
    "price": "1109",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 5G 512GB",
    "price": "995.98",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 5G 256GB",
    "price": "975.98",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 5G 512GB (Pažeista pakuotė)",
    "price": "1090",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 5G 256GB",
    "price": "970",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 5G 256GB",
    "price": "898.99",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 5G 256GB (Pažeista pakuotė)",
    "price": "898.99",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold4 5G 256GB",
    "price": "869",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 5G 256GB",
    "price": "840.2",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 5G 512GB",
    "price": "800.04",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 5G 256GB (Ekspozicinė prekė)",
    "price": "898.99",
    "error": false
  },
  {
    "name": "Samsung Galaxy Flip6 256GB",
    "price": "781",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold4 256GB (Damaged Packaging)",
    "price": "759",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 5G 512GB (Naudota prekė)",
    "price": "749",
    "error": false
  },
  {
    "name": "Samsung Galaxy S23 Plus 5G 256GB (Pažeista pakuotė)",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy Fold5 256GB (Ekspozicinė prekė)",
    "price": "",
    "error": false
  }
]